contracts/interfaces/ISummitVRFModule.sol

```diff
@@ -0,0 +1,16 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.0;
+
+
+interface ISummitVRFModule {
+    function futureBlockMined() external view returns (bool);
```

Comment:
Can we remove the extra parts of this and get it down to just getRandomNumber & setSeedRoundEndTimestamp

Reply:
setTrustedSeederAdd function is also needed as well

Reply:
just fixed